lib/Server.js

```diff
@@ -38,18 +38,19 @@ const schema = require("./options.json");
 /** @typedef {import("ipaddr.js").IPv6} IPv6 */
 /** @typedef {import("net").Socket} Socket */
 /** @typedef {import("http").IncomingMessage} IncomingMessage */
+/** @typedef {import("webpack-dev-middleware").ServerResponse} ServerResponse */
 /** @typedef {import("open").Options} OpenOptions */
 
 /** @typedef {import("https").ServerOptions & { spdy?: { plain?: boolean | undefined, ssl?: boolean | undefined, 'x-forwarded-for'?: string | undefined, protocol?: string | undefined, protocols?: string[] | undefined }}} ServerOptions */
 
 /**
  * @template Request, Response
- * @typedef {import("webpack-dev-middleware").Options<Request, Response>} DevMiddlewareOptions
+ * @typedef {import("webpack-dev-middleware").Options<IncomingMessage, ServerResponse>} DevMiddlewareOptions
```

Comment:
hm, why it was changed?

Reply:
Typescriptv5  not happy with `<Request, Response>` anymore